### DISCUSSION TITLE
Blunderbuss renders other homemade blackgunpowder weapons pointless

### DISCUSSION THREAD

**Discussion by RodriguesCIA** [cataclysmbn/Cataclysm-BN#2066] [General]
For starters, the blunderbuss itself. It is trivial to make, as indicated by the recipe, with not too bad characteristics other than weight.

![image](https://user-images.githubusercontent.com/39223166/162757616-6f3c9a0f-c746-493b-97e6-dcbf0160edbc.png)

Ammo is even easier to make, not even demanding actual gunpowder and the shot itself can be about any garbage you got, and have very good characteristics.

![image](https://user-images.githubusercontent.com/39223166/162757697-427fa4cd-88fa-4b4f-8673-204e5b9c3e06.png)

With all of this in mind, the only general advantage the other homemade guns have over the blunderbuss is better range, lighter, and lighter ammo. However, the blunderbuss one shots at critical range basic zombies, and is extremely fast to reload.

My point is, after making one for testing, I find little point in even bothering with pipe guns and flintlocks, and perhaps, even crossbows if these weren't silent. The gun is vastly overperforming on its "tech level", not even requiring heating to make and very rudimentary tools, and not even real gunpowder to fire.

**Comment by Proxiehunter** [comment]
I'm not an expert on the historical blunderbus, however _damage higher than a modern battle rifle_ seems wrong. It also seems like armor should be more effective against it. At least modern ballistic armor should be (like you would find on a Kevlar zombie or one of the military variants) although I don't think Bright Nights is capable of making that distinction.

**Comment by Coolthulhu** [comment]
A big part of it is that it was supposed to be balanced by high dispersion, but it doesn't work that well due to how dispersion actually works.
Making it use cone shot mechanics with wide shot but low damage could give it a niche, while also making it worse at armor piercing.

**Comment by chaosvolt** [comment]
Making it a cone weapon would be a good option, yeah. That said, part of the problem also stems from how the blunderbuss and pipe guns are basically the only homemade weapons that're actually obtainable at a point in the game where they'd be used, the blunderbuss even moreso because it also has ammo that's feasible to craft at that point. With the pipe guns, you likely won't be using them by the time you can craft your own ammo for them, so they only become useful if you have the desired type of ammo but have NOT found a working gun.

The lategame craftable guns meanwhile outright aren't accessible until you're at a point where you almost certainly have a gun of SOME sort by then, and likely have some way to keep it fueled for any occasions where it's vital.

These are harder issues to fix because the only real options are either try to create more situations where you'll have ammo but not the gun you want (which is hard to do without either some big overhauls to spawns), or make the homemade guns easier to access before they become obsolete.

Some hypothetical ways this could be mitigated though:
1. Guns that spawn damaged are an added hurdle to get back in reliable order since repairing them takes specific tools and some entry-level skills, so more guns that spawn well-used mean more reason to want to craft your own.
2. More occasions of gunpowder ammo showing up might help, since the lower recoil doesn't play nice with autoloading guns, and vanilla Cata doesn't have autoloading homemade guns for realism reasons so all of them SHOULD have some bare minimum level of blackpowder tolerance.
3. We could tinker with making homemade ammo in some way less reliable in a way that only affects factory-made guns. Basically exploit the idea of there being potential for looser tolerances. This does run into the same issue that handloads already have with their damage being arbitrarily lower though, in that realistically if you're getting into handloading at all you will either learn to make consistent ammo that has no reason to be inferior to factory ammo, or you're a pipebomb-maker who's eventually going to discover their true calling the hard way.
